src/PolyhedralGeometry/triangulations.jl

```diff
@@ -254,3 +254,130 @@ A polyhedron in ambient dimension 8
 function secondary_polytope(P::Polyhedron)
     return Polyhedron(Polymake.polytope.secondary_polytope(pm_object(P)))
 end
+
+@doc Markdown.doc"""
+    isregular(pts::Union{SubObjectIterator{<:PointVector}, AbstractMatrix, Oscar.MatElem},T::Vector{Vector{Vector{Int64}}})
+
+Compute whether a triangulation is regular.
+
+# Examples
+Compute whether a triangulation of the square is regular.
+```jldoctest
+julia> c = cube(2)
+A polyhedron in ambient dimension 3
+
+julia> T=[[1,2,3],[2,3,4]];
+
+julia> isregular(point_matrix(vertices(c)),T)
+true
+```
+"""
+function isregular(pts::Union{SubObjectIterator{<:PointVector}, AbstractMatrix, Oscar.MatElem},T::Vector{Vector{Int64}})
+    as_sop = SubdivisionOfPoints(pts,T)
+    isregular(as_sop)
+end
+
+
+#These all seem like natural ways to construct a subdivision of points from the data
+#  of a triangulation.
+
+
+
+@doc Markdown.doc"""
+    SubdivisionOfPoints(P::Polyhdron, Incidence::IncidenceMatrix)
+
+# Arguments
+- `P::Polyhedron`: A polyhedron whose vertices are the points of the subdivision.
+- `Incidence::IncidenceMatrix`: An incidence matrix; there is a 1 at position (i,j) if cell i contains point j, and 0 otherwise.
+
+A subdivision of points formed from points and cells made of these points. The
+cells are given as an IncidenceMatrix, where the columns represent the points
+and the rows represent the cells.
+
+# Examples
+Compute a triangulation of the square
+```jldoctest
+julia> C = cube(2);
+
+julia> Incidence = IncidenceMatrix([[1,2,3],[2,3,4]]);
+
+julia> S = SubdivisionOfPoints(C, Incidence)
+A subdivision of points in ambient dimension 2
+```
+"""
+SubdivisionOfPoints(P::Polyhedron, Incidence::IncidenceMatrix) = SubdivisionOfPoints(point_matrix(vertices(P)), Incidence)
```

Comment:
Is it necessary to apply `point_matrix` here? Maybe instead change the constructor to also eat such an iterator, analogously to e.g. `Polyhedron` or `convex_hull`.

Reply:
Actually one of your constructors below already seems to solve this issue.

Reply:
Good call! I'll remove `point_matrix`